Pod/Classes/VOKManagedObjectMapper.m

```diff
@@ -190,6 +190,11 @@ - (Class)expectedClassForObject:(NSManagedObject *)object andKey:(id)key
 
         Class managedObjectClass = objc_getClass(className);
         objc_property_t prop = class_getProperty(managedObjectClass, propertyName);
+        if ([self isKindOfClass:[VOKManagedObjectDefaultMapper class]]) {
```

Comment:
Since the `NSAssert()` statements become no-ops, I'd expect the optimization step to take the conditional out entirely, since it'd branch to nothing in both cases.